Automation Examples/020.md

```diff
@@ -0,0 +1,161 @@
+# Process
+
+Open https://www.vat19.com/, go to fun & strange stuff and download first 20 products' information (name, price, image) and save it into excel file. Template is available [here](Products%20Template.xlsx).
+
+# Steps to automate
+
+#### 1. Specify a path to your folder, open website and maximise the browser
+
+```G1ANT
+♥basePath = C:\Users\dell\Desktop\Automation Examples
+selenium.open chrome url https://www.vat19.com/
+window ✱Unique✱ style maximize
+```
+
+Use window command and part of the window title as an argument to bring it to the front and maximise.
```

Comment:
`and part of the window title as an argument` i can't see references to the window title in code, am I missing something?
![image](https://user-images.githubusercontent.com/7533717/64418522-74d2ef80-d09b-11e9-8295-fafaf4524a50.png)

Reply:
![chrome_2019-09-06_12-07-16](https://user-images.githubusercontent.com/39299658/64420164-ed877b00-d09e-11e9-8980-72988ab3d38d.png)

Reply:
hmm this feels strange, really. imho tutorial should be 100% repeatable

Reply:
Because you weren't at the main webpage, you opened Fun & Strange Stuff